### PR TITLE
Bug 1866315: Inconsistent naming in storage cluster creation

### PR DESCRIPTION
**Pull request by cloudbehl**
Signed-off-by: Ankush Behl <cloudbehl@gmail.com>

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/models.ts [hunk 1]
```diff
@@ -14,8 +14,8 @@ export const CephClusterModel: K8sKind = {
 };
 
 export const OCSServiceModel: K8sKind = {
-  label: 'OCS Cluster Service',
-  labelPlural: 'OCS Cluster Services',
+  label: 'Storage Cluster',
+  labelPlural: 'Storage Clusters',
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/models.ts:18]: Why not change abbr -> SC ?

Comment (gnehapk) [reply frontend/packages/ceph-storage-plugin/src/models.ts:18]: Even, id can be changed

**Comment by afreen23** [comment]
/lgtm cancel
Lets resolve https://github.com/openshift/console/pull/6540#discussion_r484298590

File: frontend/packages/ceph-storage-plugin/src/models.ts (hunk 1 above)
Comment (cloudbehl) [reply frontend/packages/ceph-storage-plugin/src/models.ts:18]: SC is a storage class so make sense to have `OCS` as it differentiates. 

Id is internal and is not used to display

Comment (cloudbehl) [reply frontend/packages/ceph-storage-plugin/src/models.ts:18]: @afreen23 @gnehapk ^^

**Review by gnehapk — APPROVED** [review]

File: frontend/packages/ceph-storage-plugin/src/models.ts (hunk 1 above)
Comment (afreen23) [reply frontend/packages/ceph-storage-plugin/src/models.ts:18]: Alright, I am assuming to have Storage Cluster model be aligned to its CRD spec.
Therefore changing abbr and id makes sense .
Also, we have Portworx operator which has Storage Cluster as API and it represents SC only in console.

So are we overriding intentionally the common pattern @cloudbehl ?

Comment (cloudbehl) [reply frontend/packages/ceph-storage-plugin/src/models.ts:18]: I think Abbr and id are specific to UI only and not related to CRD spec. This is used to override the default functionality. 

I'm good making it default but I found OCS more relevant & unique as an abbr. So Let me know your thoughts & I'll change it accordingly.

**Review by afreen23 — APPROVED** [review]
/lgtm

File: frontend/packages/ceph-storage-plugin/src/models.ts (hunk 1 above)
Comment (afreen23) [reply frontend/packages/ceph-storage-plugin/src/models.ts:18]: Sure, for this bug fix lets have the current change in and we can discuss over other changes offline.
Thanks!

**Pull request merged**